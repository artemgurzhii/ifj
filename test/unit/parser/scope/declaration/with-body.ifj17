scope
  function foo() as string
  end function
end scope
