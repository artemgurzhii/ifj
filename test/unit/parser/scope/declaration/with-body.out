(scope ->
  (function foo -> (id string)
))
